widgetssdk/src/main/java/com/glia/widgets/call/CallController.kt

```diff
@@ -176,20 +179,20 @@ internal class CallController(
         createNewTimerStatusCallback()
         initControllerCallbacks()
         initMessagesNotSeenCallback()
-        tryToQueueForEngagement(queueId, visitorContextAssetId, mediaType)
+        tryToQueueForEngagement(queueId)
```

Comment:
So visitorContextAssetId is no longer passed through tryToQueueForEngagement and only during enqueueForEngagement? A would like an explanation on how this works now

Reply:
The properties mentioned are saved in the ChatState, a few steps before.